class-udm-updater.php

```diff
@@ -985,7 +1002,8 @@ protected function disconnect() {
 				'data' => 'Not connected (no email found)'
 			));
 		} else {
-			$result = wp_remote_post($this->url.'&udm_action=releaseaddon&slug='.urlencode($_POST['slug']).'&e='.urlencode($options['email']),
+			$slug = (!empty($_POST['slug'])) ? $_POST['slug'] : $this->slug;
```

Comment:
@hisman What's the reason for this change (is it related to the purpose of this PR, or something else?)